src/Listener/AddClientAssets.php

```diff
@@ -21,13 +21,13 @@ class AddClientAssets
      */
     public function subscribe(Dispatcher $events)
     {
-        $events->listen(ConfigureWebApp::class, [$this, 'addAssets']);
+        $events->listen(Rendering::class, [$this, 'addAssets']);
     }
 
     /**
      * @param ConfigureClientView $event
```

Comment:
Please fix the docblock here. Apparently, it already fell out of sync before.

Reply:
Actually, since it adds nothing to the method (no description) that cannot be inferred from the method signature, please just remove it.